assembly/compliance-tool/src/test/java/org/eclipse/sw360/antenna/frontend/compliancetool/AntennaComplianceToolTest.java

```diff
@@ -0,0 +1,19 @@
+package org.eclipse.sw360.antenna.frontend.compliancetool;
+
+import org.eclipse.sw360.antenna.frontend.compliancetool.main.AntennaComplianceTool;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.Objects;
+
+public class AntennaComplianceToolTest {
+
+    @Ignore
```

Comment:
Ignoring tests?

Reply:
This test is still an ignore test. I added a comment why.

Additionally I added a test class for the Sw360Exporter that tests the execute function.